### PR TITLE
No need to use a special user to run tests

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 030171ecf345362ccb4467e4efcecc6706298954

File: dockerhub/travis-ci-image.dockerfile [hunk 1]
```diff
@@ -36,11 +36,7 @@ RUN locale-gen $LANG
...
 RUN pip install -U pip tox virtualenv
 
-RUN useradd -ms /bin/bash tester
-RUN mkdir $GNOCCHI_SRC
-RUN chown -R tester: $GNOCCHI_SRC
-
 RUN npm install s3rver --global
 
-USER tester
+RUN mkdir $GNOCCHI_SRC
```
Comment (jd) [inline dockerhub/travis-ci-image.dockerfile]: This is not even needed. Docker `-v` option will create it automatically.

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 02621419461913ae282c687ed4c4683a069de2c1

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 3a2f7c4d81e3e6003bd6b74680df3400c791c5f8

**Pull request merged**